website/content/en/status/report-2024-10-2024-12/gcc.adoc

```diff
@@ -0,0 +1,26 @@
+=== GCC on FreeBSD
+
+Links: +
+link:https://gcc.gnu.org/[GCC Project] URL: link:https://gcc.gnu.org/[] +
+link:https://gcc.gnu.org/gcc-11/[GCC 11 release series] URL: link:https://gcc.gnu.org/gcc-11/[] +
+link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gnu.org/gcc-12/[] +
+link:https://gcc.gnu.org/gcc-13/[GCC 13 release series] URL: link:https://gcc.gnu.org/gcc-13/[] +
+link:https://gcc.gnu.org/gcc-14/[GCC 14 release series] URL: link:https://gcc.gnu.org/gcc-14/[] +
+
+Contact: Lorenzo Salvadore <salvadore@FreeBSD.org>
+
+The link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=281091[exp-run] to update GCC default version from 13 to 14 is getting forward.
```

Comment:
I'd change the link text to be all of "exp-run to update GCC default version from 13 to 14".